src/+groundwater/findPhreaticSurface.m

```diff
@@ -0,0 +1,81 @@
+function [depToGWT, indxGWLay] = findPhreaticSurface(SoilVariables, KT, GroundwaterSettings)
+    %{
+        added by Mostafa, modified after Lianyu
+        This function finds the soil layer that includes the phreatic surface (saturated zone) and is used in the groundwater recharge calculations
+
+                                %%%%%%%%%% Important notes %%%%%%%%%%
+        The index order for the soil layers in STEMMUS is from bottom to top (NN:1), where NN is the number of STEMMUS soil layers,
+        which is the opposite of MODFLOW (top to bottom). So, when converting information between STEMMUS and MODFLOW, indices need to be flipped
+
+        Although the groundwater depth (variable -> "gw_Dep") is calculated already from the MODFLOW inputs, it is re-calculated again in this function for two reasons:
+        1) In this function, the groundwater depth is re-calculated, but this time is from the STEMMUS matric potential, so to make sure that ...
+            STEMMUS understood the location of the groundwater depth correctly. The re-calculated groundwater depth is stored in the "depToGWT" variable.
+        2) The depToGWT needs to be assigned to a certain soil layer. Because of that assignment, there will be a slight difference between gw_Dep  ...
+            and depToGWT (e.g. difference = 1-10 cm  based on the thickness of the soil layer that contains the depToGWT).
+
+                                %%%%%%%%%% Variables definitions %%%%%%%%%%
+        gw_Dep              depth from top soil layer to groundwater level, calculated from MODFLOW inputs
+        depToGWT            depth from top soil layer to groundwater level, calculated from STEMMUS variables
+        indxBotmLayer_R     index of the bottom layer that contains the current headBotmLayer (top to bottom)
+        indxBotmLayer       index of the bottom layer that contains the current headBotmLayer (bottom to top)
+        indxGWLay           index of the soil layer that includes the phreatic surface, so the recharge is ....
+                            extracted from that layer (index order is from top layer to bottom)
+                            Note: indxGWLay must equal to indxBotmLayer_R
+        Shh                 soil matric potential (from top layer to bottom; opposite of hh)
+        soilThick           cumulative soil layers thickness (from top to bottom)
+    %}
+
+    % Load model settings
+    ModelSettings = io.getModelSettings();
+    NN = ModelSettings.NN; % number of nodes
+    NL = ModelSettings.NL; % number of layers
+    gw_Dep = GroundwaterSettings.gw_Dep; %
```

Comment:
```suggestion
```

Reply:
Please double check. These two variables are unused.

Reply:
They were used during testing, thanks for catching this